policyengine_us/variables/household/expense/person/gambling_losses.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class gambling_losses(Variable):
+    value_type = float
+    entity = Person
+    label = "Gambling losses up to gambling winnings"
```

Comment:
What does this mean? If negative it represents winnings? If so let's flip the sign to gambling_winnings for consistency with other income variables.

Reply:
The idea is that if you have $10k in winnings that are part of your income, you can deduct losses up to $10k (without the "up to winnings" part you could gamble and lose your annual income and deduct all of it). So it's still a positive amount and referred to in the worksheet as "gambling losses" (below).

 I could also create `gambling_winnings` and `deductible_gambling_winings = min( gambling_winnings, gambling_losses )` if that would be better but didn't know if we needed that many vars for this.

<img width="949" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/33919642/a9d02e44-9c68-44eb-a376-f6355ba15e71">

<img width="551" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/33919642/9495f325-3638-438c-98f1-a179968fd2aa">

Reply:
Per our chat, let's make the three separate variables, except with `ky_deductible_gambling_losses` in case other states adopt different rules (we can revisit if we find many are consistent).